### ISSUE TITLE
npm audit security report - Prototype Pollution

### ISSUE THREAD

**Issue by ZaDarkSide** [electron/electron-packager#1146]
### Preflight Checklist
<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->

* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  * 14.2.1
* **Electron Version:**
  * 8.2.5
* **Operating System:**
  * Windows 10 (1909)

### Expected Behavior
`npm audit` command should return no vulnerabilities

### Actual Behavior
`npm audit` command returns

```
                       === npm audit security report ===                        
                                                                                
                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Low             Prototype Pollution                                           
                                                                                
  Package         yargs-parser                                                  
                                                                                
  Patched in      >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2              
                                                                                
  Dependency of   electron-packager [dev]                                       
                                                                                
  Path            electron-packager > yargs-parser                              
                                                                                
  More info       https://npmjs.com/advisories/1500                             
                                                                                
found 1 low severity vulnerability in 5101 scanned packages
  1 vulnerability requires manual review. See the full report for details.

```

**Comment by luisleee** [comment]
I found that electron-packager is using `yargs-parser v16.x`, but all of versions of `yargs-parser v16.x` have this problem and they won't fix them. Maybe we need to use v18.x of yargs-parser, butI don't know if the interface of `yargs-parser` have changed.

**Comment by nidhigupta09** [comment]
I'm facing 
  === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Prototype Pollution

  Package         yargs-parser

  Patched in      >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2

  Dependency of   react-scripts

  Path            react-scripts > webpack-dev-server > yargs > yargs-parser

  More info       https://npmjs.com/advisories/1500

found 1 low severity vulnerability in 1606 scanned packages
  1 vulnerability requires manual review. See the full report for details.


I'm facing this problem ..canyone plz help me.

**Comment by malept** [comment]
This should be fixed with version 15.0.0.

**Comment by nidhigupta09** [comment]
Thank you malept  but  15.0.0 version is not  available  
current version is 14..4.0

**Comment by malept** [comment]
It's on NPM: https://www.npmjs.com/package/electron-packager/v/15.0.0

**Comment by nidhigupta09** [comment]
it show
npm WARN tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.

**Comment by ZaDarkSide** [comment]
I believe you can safely ignore that warning. It's just a warning.

**Comment by nidhigupta09** [comment]
but why it is show "found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details"
and my reactapp is not create ,

**Comment by nidhigupta09** [comment]
it show

Installing packages. This might take a couple of minutes.
Installing react, react-dom, and react-scripts with cra-template...


> core-js@2.6.11 postinstall C:\Users\hp\my-app\node_modules\babel-runtime\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@3.6.5 postinstall C:\Users\hp\my-app\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js-pure@3.6.5 postinstall C:\Users\hp\my-app\node_modules\core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

+ react-scripts@3.4.1
+ react@16.13.1
+ cra-template@1.0.3
+ react-dom@16.13.1
added 1602 packages from 751 contributors and audited 1606 packages in 411.216s

59 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details


src and public folder is not created.

**Comment by ZaDarkSide** [comment]
Rrun `npm audit` and post the details.

**Comment by nidhigupta09** [comment]
\my-app>npm audit

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Prototype Pollution

  Package         yargs-parser

  Patched in      >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2

  Dependency of   react-scripts

  Path            react-scripts > webpack-dev-server > yargs > yargs-parser

  More info       https://npmjs.com/advisories/1500

found 1 low severity vulnerability in 1634 scanned packages
  1 vulnerability requires manual review. See the full report for details.

**Comment by ZaDarkSide** [comment]
This is a problem with `webpack-dev-server` package not `electron-packager`.
You can post them an issue about this here https://github.com/webpack/webpack-dev-server/issues

**Comment by RejithReghunathan** [comment]
=== npm audit security report ===                        
                                                                                
                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Low             Prototype Pollution                                           
                                                                                
  Package         yargs-parser                                                  
                                                                                
  Patched in      >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2              
                                                                                
  Dependency of   react-scripts                                                 
                                                                                
  Path            react-scripts > webpack-dev-server > yargs > yargs-parser     
                                                                                
  More info       https://npmjs.com/advisories/1500                             
                                                                                
found 1 low severity vulnerability in 1611 scanned packages
  1 vulnerability requires manual review. See the full report for details.

**Comment by malept** [comment]
@RejithReghunathan this is a problem with `webpack-dev-server`, not Electron Packager.

Everyone else:

This has been fixed in Electron Packager 15.0.0. I'm locking this because there hasn't been any further on-topic comments since it was resolved.
